homeassistant/components/sensor/asterisk_ami.py

```diff
@@ -0,0 +1,89 @@
+# -*- coding: utf-8 -*-
+"""Asterisk Sensors."""
+import logging
+
+from homeassistant.components import asterisk_ami as ami_platform
+from homeassistant.const import STATE_UNKNOWN
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Sensor setup.
+
+    Loads configuration and creates devices for extensions and mailboxes.
+    """
+    add_devices([AsteriskSensor(hass)])
+    for mailbox in hass.data[ami_platform.DATA_MAILBOX]:
+        add_devices([AsteriskMailbox(hass, mailbox)])
+
+
+class AsteriskSensor(Entity):
+    """Asterisk Server connection status sensor."""
+
+    def __init__(self, hass):
+        """Initialize the sensor."""
+        self._hass = hass
+        self._state = STATE_UNKNOWN
+
+    @property
+    def name(self):
+        """Return the name of the sensor."""
+        return 'Asterisk Connection'
+
+    @property
+    def state(self):
+        """Return the state of the sensor."""
+        return self._state
+
+    @property
+    def should_poll(self):
+        """Return the polling mode of teh sensor."""
```

Comment:
Typo: `teh`

Also, polling is the default. thus `should_poll()` can be removed.